tests/integration/src/rust_masm_tests/abi_transform/tx_kernel.rs

```diff
@@ -50,7 +46,11 @@ fn test_get_inputs(test_name: &str, expected_inputs: Vec<Felt>) {
     test.expect_masm(expect_file![format!("../../../expected/{artifact_name}.masm")]);
 
     let vm_program = test.vm_masm_program();
-    // let vm_out = execute_vm_tracing(&vm_program, &[]).unwrap();
+
+    let exec = MidenExecutor::new(vec![]);
+    let trace = exec.execute(&vm_program);
+    let vm_out = trace.into_outputs();
+    dbg!(&vm_out);
```

Comment:
I suppose the plan is to use `trace.read_from_rust_memory` to compare with expected inputs, right? Just like in the `test_blake3_hash` test.

Reply:
Yeah exactly, we'll get some return values on the operand stack, but will need to use `read_from_rust_memory` to validate the contents of the returned vector.